docs/casestudies/unionbigdata.md

```diff
@@ -1,95 +1,89 @@
 ---
-title: 数之联在 BOE 的 I3Plat 建设中使用 Karmada
+title: UnionBigData's Utilization of Karmada in the Construction of I3Plat at BOE
 ---
 
-## 行业背景
+## 1. Industry Background
```

Comment:
```suggestion
## Industry Background
```
Not very necessary to maintain both `##` and `1.`, which have the same meaning.

Reply:
I'll remove the numbers.